packages/react-scripts/config/webpack.config.js

```diff
@@ -254,6 +254,11 @@ module.exports = function (webpackEnv) {
       level: 'none',
     },
     optimization: {
+      splitChunks: isEnvDevelopment
+        ? undefined
+        : {
+            chunks: 'all',
+          },
```

Comment:
Maybe better will be write smth like this?

```
splitChunks: isEnvProduction && { chunks: 'all' },
```

Reply:
@tednaaa In this case, the `splitChunks` key will be set to `false`, thereby overwriting the default configuration of the `SplitChunksPlugin`